server/README.md

```diff
@@ -506,16 +448,15 @@ npm run lint
 - **PUT /api/v1/recipes/:id**
 
   Alters the properties, defined in the request body, on the recipe with the
-  provided id. The request body must contain all recipe properties except the id.
-  To update only specific field you need to define these fields in the fields query parameter.
-  If a fields query prameter is defined, only these fields will be updated.
+  provided id.
+  If the request body only conatains single fields of a recipe, only these fields
+  will be updated. This is true for the first level of properties. E.g. if you update
+  the steps of a recipe and you only provide a single step all remaining steps are
+  removed since there are no data in the request.
```

Comment:
> if you update the steps of a recipe and you only provide a single step all remaining steps are removed since there are no data in the request.

what if we use the stepNumber as identification? or add an objectId? That's makes it possible to update a single recipe step!? what did you think?

Reply:
There is already a _id for recipe steps. I means it is not possible with the current implementation. And I wanted to see if we really need to update single steps in the future before I implement this.

Reply:
Okay.